src/main/scala/org/ua/ilm/datamining/crawlers/youtube/CrawlerChannels.scala

```diff
@@ -0,0 +1,16 @@
+package org.ua.ilm.datamining.crawlers.youtube
+
+import com.google.api.services.youtube.YouTube
+import com.google.api.services.youtube.model.ChannelListResponse
+
+class CrawlerChannels(val youTube: YouTube, idChannels: String,
+                      partChannels: String = "contentDetails,snippet,brandingSettings,contentOwnerDetails,localizations,statistics,status,topicDetails") {
+
+  private val part = partChannels
+  private val listIdChannels = idChannels
+  private val youtube = youTube
```

Comment:
let's initialize it in class body. I think there is no need in pathing via method args.
private val youtube = super().createYT() or even make this variable protected and move to basic crawler